### PR TITLE
Toolbar and panel background color

### PR DESCRIPTION
**Pull request by Siyubu**
1. Changed toolbar background color
2. Changed panel menu background color

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,21 @@ described in section ["Build with Maven"](#build-with-maven).
 
 ### Preparing the tools
 
+### Chocolatey
```
Comment (oscerd) [inline README.md]: This is not related to your PR. It seems you mixed things up

File: antora-ui-camel/package.json [hunk 2]
```diff
@@ -72,6 +72,7 @@
     "pnp": true
   },
   "dependencies": {
-    "algoliasearch": "^4.0.3"
+    "algoliasearch": "^4.0.3",
```
Comment (oscerd) [inline antora-ui-camel/package.json]: This too, it's not related to your PR.

**Comment by Siyubu** [comment]
> I don't think this is correct: there are commits from other PR inside and conflicts. Also what is the reason to change the color of the navbar, is there an issue for this? Improvements are ok, bug fixes too, opening PRs just for opening PRs, it's not.

I was working on Improving color contrast which is camel-14700 issue

**Comment by oscerd** [comment]
The yarn.lock doesn't seem correct anyway

**Comment by zregvart** [comment]
Same as in #235, you either need to work from a single branch and on one pull request at a time, or work from multiple branches for multiple pull requests.

Please separate the changes into multiple branches and create pull request for each change separately.

**Comment by Siyubu** [comment]
Thanks to constantly give me feedback. As it was my first time to
contribute on open sources and I was not familiar with git flow it isn't
been easy to me. Now, I would want to ask how to open issue on Jira.
While navigating
on apache website I saw pages that don't have toolbars and right navigation
bar such as support and user stories from community
*Solange Iyubu*

Student

Carnegie Mellon University in Africa

Tel: +250 785 562 715

**Comment by Siyubu** [comment]
I finally found how to open an issue. however, I can't access camel
websites .html files to be able to add toolbar breadcrumbs on support file.
I would want your help
*Solange Iyubu*

Student

Carnegie Mellon University in Africa

Tel: +250 785 562 715

**Comment by zregvart** [comment]
@Siyubu I understand that you encounter difficulties in contributing, a lot of this is new and challenging. Honestly we did not expect this many applicants to be interested in contributing and we it's been a challenge to keep up and help everyone that needs help. I can direct you to the [README](https://github.com/apache/camel-website/blob/master/README.md) file in the project that explains how to get started, [contributing guide](https://camel.apache.org/manual/latest/contributing.html) on the website and [getting started](https://help.github.com/en/github/getting-started-with-github) with GitHub.

**Comment by Siyubu** [comment]
Hello
I tried to read the documents above and follow instructions. however IO
can't building  camel using maven, I am getting plugins error and when I
try to debug using mvn -x I am getting the information attached on this
email. I would want  your help
*Solange Iyubu*

Student

Carnegie Mellon University in Africa

Tel: +250 785 562 715

**Comment by zregvart** [comment]
@Siyubu I see that you're trying to send emails with attachments here. This will not work as you're replying to GitHub sent email and only text of your reply is sent here.

To discuss development issues and as instructed in the [contributing guide](https://camel.apache.org/manual/latest/contributing.html) please subscribe to the [developer mailing list](https://camel.apache.org/community/mailing-list/) and ask questions there.

It is more helpful if you paste the text of the error you experience to an on-line service such as https://gist.github.com/ or https://pastebin.com/. Please do not send screenshots of text.

We're also conversing on a closed pull request at https://github.com/apache/camel-website/pull/253 please bare that in mind.

**Comment by Siyubu** [comment]
Okay, I am sorry for the inconveniences
*Solange Iyubu*

Student

Carnegie Mellon University in Africa

Tel: +250 785 562 715
